module/VuFind/src/VuFind/ILS/Driver/Alma.php

```diff
@@ -571,7 +571,7 @@ public function getAccountBlocks($patron)
      * @param array  $fulfillmentUnits An array of fulfillment units with all its
      *                                 locations.
      *
-     * @return string|NULL              Null if the location was not found or a
+     * @return string|null              Null if the location was not found or a
```

Comment:
This shorter syntax is used more commonly in the code base:

```suggestion
     * @return ?string                  Null if the location was not found or a
```

...it might also be nice to unwrap the description here, since it really doesn't need to take up three lines.

Reply:
done